typed_ast/ast27.py

```diff
@@ -207,10 +207,13 @@ def get_docstring(node, clean=True):
         raise TypeError("%r can't have docstrings" % node.__class__.__name__)
     if node.body and isinstance(node.body[0], Expr) and \
        isinstance(node.body[0].value, Str):
+        docstring = node.body[0].value.s
         if clean:
             import inspect
-            return inspect.cleandoc(node.body[0].value.s)
-        return node.body[0].value.s
+            if isinstance(docstring, str):
+                return inspect.cleandoc(docstring)
+            return inspect.cleandoc(docstring.decode()).encode()
```

Comment:
This is possibly unsafe because we don't know what encoding the string is in. Honestly I'm not sure it's worth changing anything here.

Reply:
I thought PEP 263 meant this would be UTF-8. But agreed that this is unlikely to help anyone.

Reply:
I don't think Python 2 privileged UTF-8, it only became the default in Python 3.